roles/vdm/tasks/postgres/pg_config.yaml

```diff
@@ -0,0 +1,57 @@
+- name: pg_config - Check for storage info ConfigMap
+  block:
+  - name: set facts 
+    set_fact:
+      dac_crunchy_storage_cm_found: false
+      v4_crunchy_present: false
```

Comment:
We have both `found` and `present` in 2 facts. Lets just use `found` as the flag item. Also wondering why the `v4_crunchy_present` value is here if the logic is looking for the config map about storage.

Reply:
I was setting v4_crunchy_present to false here to define an acceptable default for the fact. I can remove it here and add a when is defined prior to testing for true or false in postgres-instance.